### DISCUSSION TITLE
RED-OPS Cross-seed

### DISCUSSION THREAD

**Discussion by khlaus** [RogueOneEcho/caesura#94] [Support]
I’m experiencing a strange issue, I transcode from 24-bit to 16-bit on RED. After that, I transcode the same files using a torrent from OPS that I uploaded with redcurry.(I'm not sure what it's doing when it already finds the first transcode there, does it skip?). 

 Everything works fine for a while, and I even recheck the torrent, which completes successfully at 100%.However, if I recheck the torrent after about 5-10 minutes, it shows as not working anymore. This typically happens with the first torrent I processed (the one I transcoded via RED). When I open the destination folder, the files are still there, and I’m not sure what’s causing the issue.

**Comment by RogueOneEcho** [comment]
Ok, this sounds rather complex. Can you confirm I've got this right?

1) You download a 24 bit source from RED
2) You transcode to 16 bit using `caesura`
3) You upload the 24 bit source from RED to OPS using `redcurry`
4) You transcode again to 16 bit using `caesura` (to clarify, what actually happens is the application finds the already transcoded `.torrent` file in the output directory, verifies the transcoded files exist and haven't changed, then simply create a `.ops.torrent` file without re-transcoding the files).
5) You upload the 16 bit to OPS
6) You add the `.ops.torrent` to your client and it successfully validates that 100% of files exist.
7) After 5-10 minutes your torrent client shows an error that the 16 bit RED transcode are no longer valid, but somehow the 16 bit OPS transcode are fine.

**Reply by khlaus** [reply]
Exactly,I know it sounds extremely weird. But this happens every single time. I see no reason why the 16 bit ops shouldn't just work. If I don't use caesura to make the 16bit for ops, and I do it manually then it works. I basically made a .bat script that transcodes and uploads everything on both sites no matter the source, that's my intention.
- you said RED twice at 7, so RED(first transcode) doesn't work anymore but OPS(2nd) does.

**Reply by RogueOneEcho** [reply]
Apparently I got sidetracked while writing that and forgot to ask:

1) What torrent client are you using?

2) If you `force recheck` immediately after the `.ops.torrent` is added to the client is it successful, basically is there something significant about the 5-10 minute delay?

3) How are you adding the torrent to the client, is it via autoadd, are you doing it manually, or is it an injection script?

4) Are you able to manually verify/check the torrents?

Using [imdl](https://github.com/casey/intermodal) try running the following against the `.ops.torrent` and `.red.torrent`, and even better if you can track down the correct `.torrent` (by its infohash)  in `BT_backup`  directory of qBit or `deluge/state` of deluge do those verify?

```
imdl torrent verify [path to .torrent]
```

**Comment by khlaus** [comment]
1.Qbit
2.It is successful.
3.Manually(with category and automanagement on--but the category is set in the same folder, the files aren't moved)
4. The first one is RED
![Image](https://ptpimg.me/dt6g78.png)


Edit: I've checked this one after writing here, and now it was successful, lol. I will keep checking after people finish downloading it from me.
Edit2: So this one kept working, it's weird because I didn't change anything in the way I'm doing things. Let me try a bunch more, maybe it was something wrong on my end and I'm wasting your time for nothing. I will come back with an update.

**Reply by RogueOneEcho** [reply]
I suspect in this screenshot the IMDL command was failing for you there because it's looking for the torrent content in the same directory as the torrent file. I should have included the [`--content` argument](https://imdl.io/book/commands/imdl-torrent-verify.html)

```
imdl torrent verify [path to .torrent]  --content [path to torrent content]
```

Glad to hear it's working for you now. I'll close this for now. but don't hesitate to ping me a message if it fails again and you suspect it's a `caesura` issue.

**Comment by khlaus** [comment]
`2024-11-26 14:34:32.166 DBUG * Skipping Blaucrowd Surfer - Kommen und Gehen [2024] [WEB FLAC 24BIT] as it doesn't have an id,`
I get this very often when using batch, not always though.
And if I do it with the transcode command giving the url or id as argument it works.

**Reply by RogueOneEcho** [reply]
Which version are you using? And is this specific to RED torrents since the domain change?

 I suspect updating to https://github.com/RogueOneEcho/caesura/releases/tag/v0.22.0 (released two hours ago) will fix it for you.

The batch command works by reading the `comment` field of the `.torrent` file and parsing the torrent id from the url. This used to be fine but since RED has changed from `.ch` to `.sh` older versions it no longer works. 

Related #92

**Reply by khlaus** [reply]
I see, so if I make the torrent then the comment field is empty, and I have to redownload the torrent from the website in order for it to work. It's good that I have a solution for this, thank you, and thank for your work, I really appreciate this app!
